### ISSUE TITLE
Only cropped images are shown.

### ISSUE THREAD

**Issue by wilfriedjonker** [JimBobSquarePants/UmbracoFileSystemProviders.Azure#81]
Best,

At the moment I'am preparing a presentation for the company I work for regarding Umbraco in Azure. I have attended a [workshop](https://www.umbx.live/microsoft-azure-with-umbraco/) from Jeavon Leopold regarding this subject. Now I want to implement it on my own I have a problem with viewing non-cropped images. Cropped images ar not a problem.

Situation
* media folder in Azure web app is empty.
* The images are stored in the container http://beartheload.blob.core.windows.net/media/.
* The image is available in the container (e.g. http://beartheload.blob.core.windows.net/media/1044/14441035391_7ee1d0d166_h_darken.jpg)
* The image is not available in the web app (e.g. http://beartheload.azurewebsites.net/media/1044/14441035391_7ee1d0d166_h_darken.jpg)
* If the image is cropped, it is working (e.g. http://beartheload.azurewebsites.net/media/1044/14441035391_7ee1d0d166_h_darken.jpg?height=63255&width=205).
* UmbracoFileSystemProviders.Azure is version 0.5.8-beta
* ImageProcessor.Web.Config is version 2.3.0.

Relevent web.config parts : 
[web-config-snippet.txt](https://github.com/JimBobSquarePants/UmbracoFileSystemProviders.Azure/files/1017954/web-config-snippet.txt)

The FileSystemProvider.config file
[FileSystemProvider.txt](https://github.com/JimBobSquarePants/UmbracoFileSystemProviders.Azure/files/1017926/FileSystemProvider.txt)

I hope someone is able to help me solve this problem.

**Comment by Jeavon** [comment]
@wilfriedjonker what do you have in your /media/web.config ?

**Comment by wilfriedjonker** [comment]
Jeavon,

Thanks for the quick response, it put me on the right track. I had deleted the file, so it was not uploaded. After uploading the original web.config file it was still not working, I found out the following line was missing :
<add name="StaticFileHandler" path="*" verb="*" preCondition="integratedMode" type="System.Web.StaticFileHandler"/>

Now the images are shown correct. This is the [working Web.config.txt](https://github.com/JimBobSquarePants/UmbracoFileSystemProviders.Azure/files/1018499/Web.config.txt)
